sdk/ai/Azure.AI.Projects/tests/Azure.AI.Projects.Tests.csproj

```diff
@@ -1,35 +1,37 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
-
     <!-- We don't care about XML doc comments on test types and members -->
     <NoWarn>$(NoWarn);CS1591</NoWarn>
   </PropertyGroup>
 
-  <!-- Reference the Client Library -->
+  <!-- Reference Foundry libraries via project -->
   <ItemGroup>
     <ProjectReference Include="$(AzureCoreTestFramework)" />
     <ProjectReference Include="..\src\Azure.AI.Projects.csproj" />
+    <ProjectReference Include="..\..\Azure.AI.Projects.OpenAI\src\Azure.AI.Projects.OpenAI.csproj" />
+    <ProjectReference Include="..\..\Azure.AI.Agents.Persistent\src\Azure.AI.Agents.Persistent.csproj" />
+    <ProjectReference Include="..\..\Azure.AI.Inference\src\Azure.AI.Inference.csproj" />
   </ItemGroup>
-  
+
   <ItemGroup>
-    <PackageReference Include="Azure.AI.Agents.Persistent" VersionOverride="1.0.0" />
-    <PackageReference Include="Azure.AI.OpenAI" VersionOverride="2.2.0-beta.4" />
-    <PackageReference Include="Azure.AI.Inference" />
-    <PackageReference Include="NUnit" />
-    <PackageReference Include="NUnit3TestAdapter" />
+    <PackageReference Include="Azure.AI.OpenAI" />
+    <PackageReference Include="NUnit" VersionOverride="4.4.0" />
```

Comment:
Gotta remove the VersionOverrides.   Because these are part of ClientModel.TestFramework, you'll get the right versions transitively and do not need a direct reference.